sig-storage/annual-report-2021.md

```diff
@@ -0,0 +1,164 @@
+  * PVC created by statefulset will not be auto removed (sig-apps): https://github.com/kubernetes/enhancements/pull/1915
+  * Volume expansion for Stateful sets (sig-apps): https://github.com/kubernetes/enhancements/pull/1848
+  * Enable User namespaces in kubelet so UIDs get shifted (sig-node): https://github.com/kubernetes/enhancements/pull/2101
+ContainerNotifier (sig-node): https://github.com/kubernetes/enhancements/pull/1995
+  * User id ownership in configmaps and secrets (sig-auth)
+
+What areas and/or subprojects does the group need the most help with?
+* There are features that we co-owned with sig-node, sig-apps, etc. We would need help from other SIGs to move things forward.
+  * Features co-owned with sig-node:
+    * ContainerNotifier: https://github.com/kubernetes/enhancements/pull/1995
+    * Volume health: https://github.com/kubernetes/enhancements/pull/2286
+  * Features co-owned with sig-apps:
+    * PVC created by statefulset will not be auto removed: https://github.com/kubernetes/enhancements/pull/1915
+    * Volume expansion for statefulset: https://github.com/kubernetes/enhancements/pull/1848
+  * Features co-owned with sig-scheduling:
+    * Prioritization on volume capacity (sig-scheduling): https://github.com/kubernetes/enhancements/pull/1862
+
+* There are also areas where we need sig-architecture’s help. For example, VolumeSnapshot is already GA and it is a core feature in sig-storage developed using CRDs. We have documented and blogged that components such as the snapshot CRDs and snapshot controller need to be deployed by Kubernetes distros, however, not every distro is installing them. This is a problem with any feature developed using CRD. We hope sig-architecture can help us resolve this problem.
+
+What's the average open days of a PR and Issue in your group? / what metrics does your group care about and/or measure?
+
+According to the following stats on 1/25/2021, it shows the 7 day MA for a PR Time to Approve and Merge in sig-storage repository group is:
+Open to lgtm: avg 3.55 days, max 14.11 week
+Lgtm to approve: avg 12.60 hour, max 3.92 week
+Approve to merge: avg 1.03 hour, max 1.14 day
+85% open to lgtm: avg 1.4 week, max 14.11 week
+85% lgtm to approve: avg 17.11 hour, max 3.92 week
+85% approve to merge: avg 1.2 day, max 1.86 week
+https://k8s.devstats.cncf.io/d/44/pr-time-to-approve-and-merge?orgId=1&var-period=d7&var-repogroup_name=SIG%20Storage&var-apichange=All&var-size_name=All&var-kind_name=All
+
+Age of 7 day MA of issues by sig-storage repository group on 1/25/2021:
+Median time to close issue: Min 0.93 hour, Max 23.55 week, Avg 3.30 week
+https://k8s.devstats.cncf.io/d/15/issues-age-by-sig-and-repository-groups?orgId=1&var-period=d7&var-repogroup_name=SIG%20Storage&var-sig_name=All&var-kind_name=All&var-prio_name=All
+
+Suggestions on how to improve the PR/Issue velocity will be beneficial.
```

Comment:
Some suggestions I have:
- increase the reviewer pool
- regular issue triage meetings

Reply:
Added.